requirements/requirements-dev.txt

```diff
@@ -17,13 +17,13 @@ attrs==19.1.0             # via automat, black, flake8-bugbear, hypothesis, json
 automat==0.7.0            # via twisted
 autopep8==1.4.4
 babel==2.7.0
-bcrypt==3.1.6             # via matrix-synapse
+bcrypt==3.1.7             # via matrix-synapse
```

Comment:
Hmm there are way too many dependency changes. Why are you introducing them in this PR? Perhaps you guys changed something in the way dependencies are handled while I was gone?

Reply:
I didn't know `requirements/dep upgrade` can take a package name.  Will fix.